src/pending_message.cpp

```diff
@@ -132,7 +132,7 @@ std::string PendingMessage::ApplyTextInsertingCommands(std::string input, uint32
 		// Fast path - no substitutions occured, so just move the input into the return value.
 		output = std::move(input);
 	} else {
-		output.append(start_copy, end);
+		output.append(start_copy, end - start_copy);
```

Comment:
No side effects but doesn't this substraction change the API?

Reply:
No, it is same.
https://www.cplusplus.com/reference/string/string/append/

Reply:
Verified this now also in my IDE: This picks a different overload... C++ is a mess :)